testing/unit_tests/cases_loader.py

```diff
@@ -0,0 +1,242 @@
+from pathlib import Path
+
+import torch
+
+from testing.unit_tests.t_utils import __data_testing_dir__, path_repo_root
+from ivadomed.keywords import LoaderParamsKW, ModelParamsKW, TransformationKW
+
+
+def get_microscopy_loader_parameters():
+    default_loader_parameters = {
+        LoaderParamsKW.PATH_DATA: [str(Path(__data_testing_dir__, "microscopy_png"))],
+        LoaderParamsKW.BIDS_CONFIG: f"{path_repo_root}/ivadomed/config/config_bids.json",
+        LoaderParamsKW.TARGET_SUFFIX: [["_seg-myelin-manual", "_seg-axon-manual"]],
+        LoaderParamsKW.EXTENSIONS: [".png"],
+        LoaderParamsKW.ROI_PARAMS: {"suffix": None, "slice_filter_roi": None},
+        LoaderParamsKW.CONTRAST_PARAMS: {"contrast_lst": []}
+    }
+
+    return default_loader_parameters
+
+
+def case_bids_df_microscopy_png():
+    loader_parameters = get_microscopy_loader_parameters()
+
+    return loader_parameters
+
+
+def case_bids_df_anat():
+    loader_parameters = {
+        LoaderParamsKW.PATH_DATA: [__data_testing_dir__],
+        LoaderParamsKW.TARGET_SUFFIX: ["_lesion-manual"],
+        LoaderParamsKW.EXTENSIONS: [],
+        LoaderParamsKW.ROI_PARAMS: {"suffix": "_seg-manual", "slice_filter_roi": None},
+        LoaderParamsKW.CONTRAST_PARAMS: {"contrast_lst": ["T1w", "T2w"]}
+    }
+
+    return loader_parameters
+
+
+def case_bids_df_multi():
+    loader_parameters = get_microscopy_loader_parameters()
+    loader_parameters.update({
+        LoaderParamsKW.PATH_DATA: [__data_testing_dir__, str(Path(__data_testing_dir__, "microscopy_png"))],
+        LoaderParamsKW.TARGET_SUFFIX: ["_seg-manual", "seg-axon-manual"],
+        LoaderParamsKW.EXTENSIONS: [".nii.gz", ".png"],
+        LoaderParamsKW.CONTRAST_PARAMS: {"contrast_lst": ["T1w", "T2w", "SEM"]}
+    })
+
+    return loader_parameters
+
+
+def case_bids_df_ctscan():
+    loader_parameters = {
+        LoaderParamsKW.PATH_DATA: [str(Path(__data_testing_dir__, "ct_scan"))],
+        LoaderParamsKW.BIDS_CONFIG: f"{path_repo_root}/ivadomed/config/config_bids.json",
+        LoaderParamsKW.TARGET_SUFFIX: ["_seg-manual"],
+        LoaderParamsKW.EXTENSIONS: [".nii.gz"],
+        LoaderParamsKW.ROI_PARAMS: {"suffix": None, "slice_filter_roi": None},
+        LoaderParamsKW.CONTRAST_PARAMS: {"contrast_lst": ["ct"]}
+    }
```

Comment:
I would still recommend to load from the `get_microscopy_loader_parameters()` for future compatibility reason. 

In the event some new future keys are missed here from being included during this loader_parameter's dictionary definition. Better safe than sorry.